### PR TITLE
cli, documentation: improve quick start (bug 1802211)

### PR DESCRIPTION
**Pull request by zzzeid**
- add command to search for users
- elaborate in quick start section of documentation
- add quick start comment on top of yaml file

### CODE REVIEW

File: documentation/index.rst [hunk 1]
```diff
@@ -34,11 +34,86 @@ Installing pre-releases gives you early access to features or bug fixes, and hel
...
+#. Add a new entry with ``bmo_id`` as the key, and the ID as the value
+#. Save and exit
+#. Run ``mots export-and-clean`` to process the new changes
+#. Commit your changes and submit your revision for review
+
+.. tip::
+   You can find the Bugzilla ID of a user by running ``mots user search <email>``.
+
+.. note::
+   Step 4 will fetch additional user information from the Bugzilla API and update the
```
Comment (cgsheeh) [inline documentation/index.rst:85]:
```suggestion
   Running ``mots export-and-clean`` will fetch additional user information from the Bugzilla API and update the
```

File: mots.yaml [hunk 2]
```diff
@@ -1,6 +1,10 @@
+%YAML 1.2
```
Comment (cgsheeh) [inline mots.yaml:1]: We should have the MPL here, and in m-c if it isn't already.

**Review by cgsheeh — CHANGES_REQUESTED** [review]
Meta: IMO you should use proper sentences in this documentation, specifically periods at the end of the bullet points.

File: src/mots/bmo.py [hunk 3]
```diff
@@ -61,7 +61,7 @@ def get_users_by_ids(self, ids: list[str]):
         result = response.json()
         return {u["id"]: u for u in result["users"]}
 
-    def get_match(self, match: str):
+    def get_matches(self, match: str) -> list[dict]:
```
Comment (cgsheeh) [inline src/mots/bmo.py:64]: Guess this was unused before, heh.

Would be nice to have a test for it.

**Review by cgsheeh — APPROVED** [review]

**Review by cgsheeh — APPROVED** [review]

File: src/mots/bmo.py (hunk 3 above)
Comment (zzzeid) [reply src/mots/bmo.py:64]: Yes, it was left as a remnant from an early implementation. It makes more sense to search by IDs for various reasons.

**Pull request merged**